babel.config.json

```diff
@@ -1,9 +1,13 @@
 {
-    "presets": ["@babel/preset-env", "@babel/preset-react"],
+    "presets": [
+        ["@babel/preset-env", {"useBuiltIns": "usage", "corejs": "3.18"}],
+        "@babel/preset-react"
+    ],
     "plugins": [
         ["@babel/plugin-proposal-decorators", {"legacy": true}],
-        "@babel/plugin-proposal-object-rest-spread",
-        "@babel/plugin-transform-flow-strip-types",
-        ["@babel/plugin-proposal-class-properties", {"loose": true}]
-    ]
+        "@babel/plugin-transform-flow-strip-types"
+    ],
+    "assumptions": {
```

Comment:
this prevents the annoying `@babel/plugin-proposal-class-properties` warnings during build. we already added this on the `2.3` branch in #5993 🙂